test/JDBC/upgrade/master/preparation/BABEL-4206-vu-prepare.sql

```diff
@@ -0,0 +1,57 @@
+CREATE LOGIN babel_4206_vu_prepare_login1 WITH PASSWORD = 'abc';
```

Comment:
We need to test it in both single-db and multi-db migration mode. Because it makes difference for dbo, guest, db_owner user and dbo schema's physical name.

Reply:
Correct, currently we don't have infrastructure to test single-db dump/restore but I am tracking this separately (BABEL-4053) and address this there.